hapic/data.py

```diff
@@ -32,5 +33,14 @@ def get_content_disposition_header_value(self) -> str:
         if self.as_attachment:
             disposition = 'attachment'
         if self.filename:
-            disposition = '{}; filename="{}"'.format(disposition, self.filename)
+            # INFO - G.M - 2018-10-26 - deal correctly with unicode filename
+            # see rfc6266 for more info.
+            ascii_filename = self.filename.encode('ascii', 'replace').decode()
```

Comment:
non ascii characters are replaced by `? ` in ascii_filename:

```
>>> '€'.encode('ascii', 'replace').decode()
'?'
```
It will be compatible with header ?

Reply:
i just don't understand the issue.

Reply:
@lebouquetin Ascii named is only for legacy browser support, `filename` allow only latin-1 character whereas `filename*` support utf-8 encoding, so to deal with this we need to provide both to support most case.

Reply:
We could provided latin-1 support in `filename` but as read here, this seems to be not well supported and depreacated pratices since  RFC 7230:   http://www.bortzmeyer.org/6266.html .

Reply:
@buxx Unclear for "?" char but ascii is allowed and "?" is a valid value for "Tk" header : https://en.wikipedia.org/wiki/List_of_HTTP_header_fields#Field_values
So i imagine that there is no problem with "?" here.

Reply:
@lebouquetin @inkhey Ok. For me the code is ok with these informations. @lebouquetin are you ok to ?

Reply:
not yet. @inkhey you have a variable `urlencoded_unicode_filename = quote(self.filename)`. urlencode means for example replace space with %20. It looks like its not the case here (and it's fine), so the name should be something like `quoted_filename` instead of `urlencoded_unicode_filename`, doesnt it?

Reply:
@lebouquetin space are clearly replaced with %20 like any other character which may cause trouble in url. So urlencode is probably correct naming.
```
>>> from urllib.parse import quote
>>> quote(' ')
'%20'
```

Reply:
If the encoding is required by the RFC, then add the explanation in a comment

Reply:
By the way, I misunderstood the code because *quote* origin was out of context 

What do you think about importing and using the function like this:

Import : `import urllib.parse`
Usage : `urllib.parse.quote(xxx)`

?

Reply:
@inkhey what about my last comment.

Reply:
done.